configure_tls_protocol_version_and_ciphers.md

```diff
@@ -0,0 +1,18 @@
+# Configure TLS Protocol Version and Ciphers
+
+## SDK
```

Comment:
I recommend installing a markdown linter. It'll tell you to put newlines around headers.